package.json

```diff
@@ -20,27 +20,38 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "style": "style/index.css",
+  "sideEffects": [
+    "style/*.css"
+  ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/tslaton/jupyterlab-recents.git"
+    "url": "https://github.com/NERSC/jupyterlab-recents.git"
   },
   "scripts": {
     "build": "tsc",
-    "clean": "rimraf lib && rimraf tsconfig.tsbuildinfo",
```

Comment:
`rimraf` is like `rm`. It can take a list of elements to be deleted. By passing multiple elements, you avoid starting multiple times the process.

FYI: I changed the line to match what is in the [current extension template](https://github.com/jupyterlab/extension-cookiecutter-ts/blob/f33180f1c82bd6b310280d002d8a88f77005fcb9/%7B%7Bcookiecutter.extension_name%7D%7D/package.json#L39).